pages/settings/index.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <v-container>
-    <v-row>
-      <v-col cols="12" offset-lg="1" md="5" lg="4" class="pt-0 pb-4">
+    <v-row class="pt-4">
+      <v-col cols="12" offset-lg="1" md="5" lg="4" class="pt-4 pb-4">
```

Comment:
```suggestion
      <v-col cols="12" offset-lg="1" md="5" lg="4" class="py-4">
```

Reply:
Thanks. I did not that existed :) I learned a new thing today